pkg/aggregator/context_resolver.go

```diff
@@ -38,7 +38,7 @@ func newContextResolver() *contextResolver {
 	return &contextResolver{
 		contextsByKey: make(map[ckey.ContextKey]*Context),
 		keyGenerator:  ckey.NewKeyGenerator(),
-		tagsBuffer:    util.NewTagsBuilder(),
```

Comment:
Later in this file (not changed in this PR), `trackContext` creates a Context, which contains `Tags []string`.  Would it make sense to instead store a HashedTags instance there?

Reply:
I don't think so, we only need the tags' hashes for the context key. Once the tags are stored in the context, we will basically not look at them again until we serialize metrics for the intake.